config/initializers/stripe.rb

```diff
@@ -1,9 +1,3 @@
-Rails.configuration.stripe = {
-  publishable_key: ApplicationConfig["STRIPE_PUBLISHABLE_KEY"]
-}
-
-Stripe.api_key = ApplicationConfig["STRIPE_SECRET_KEY"]
-
 if Rails.env.development? && Stripe.api_key.present?
```

Comment:
we should move this in `initialize_stripe`, as `Stripe.api_key` is not set here this code will never be executed